### ISSUE TITLE
Example doesn't work on deno deploy playground

### ISSUE THREAD

**Issue by btwiuse** [asos-craigmorten/opine-http-proxy#14]
# Issue

Setup:

- Deno Version: Deno deploy
- v8 Version: Deno deploy
- Typescript Version: Deno deploy
- Opine HTTP Proxy Version: 3.0.1
- Opine Version: 2.0.2

## Details

Deno deploy uses port 8000 by default.

The following example works locally,
```
import { proxy } from "https://deno.land/x/opineHttpProxy@3.0.1/mod.ts";
import { opine } from "https://deno.land/x/opine@2.0.2/mod.ts";

const app = opine();

app.use(proxy("https://github.com/cmorten/opine-http-proxy"));

app.listen(8000);
```
but when I deploy it to deno playground, it returns the following error:
```
BadRequestError: Bad Request
    at decodeContent (https://deno.land/x/opine@2.0.2/src/middleware/bodyParser/read.ts:66:15)
    at read (https://deno.land/x/opine@2.0.2/src/middleware/bodyParser/read.ts:18:21)
    at https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:40:35
    at new Promise (<anonymous>)
    at getBody (https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:40:12)
    at reqBody (https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:50:41)
    at createRequestInit (https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:66:24)
    at buildProxyReqInit (https://deno.land/x/opineHttpProxy@3.0.1/src/steps/buildProxyReqInit.ts:6:28)
    at async Layer.handle [as handle_request] (https://deno.land/x/opine@2.0.2/src/router/layer.ts:49:9)
```

I'm not sure if this is caused by some limitations of deno deploy sandbox.

It would be cool if we can use opine-http-proxy from official deno playground.

I have made it public at https://dash.deno.com/playground/opine-play

**Comment by reggi** [comment]
same:

```
BadRequestError: Bad Request
    at decodeContent (https://deno.land/x/opine@2.0.2/src/middleware/bodyParser/read.ts:66:15)
    at read (https://deno.land/x/opine@2.0.2/src/middleware/bodyParser/read.ts:18:21)
    at https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:40:35
    at new Promise (<anonymous>)
    at getBody (https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:40:12)
    at reqBody (https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:50:41)
    at createRequestInit (https://deno.land/x/opineHttpProxy@3.0.1/src/requestOptions.ts:66:24)
    at buildProxyReqInit (https://deno.land/x/opineHttpProxy@3.0.1/src/steps/buildProxyReqInit.ts:6:28)
    at async Layer.handle [as handle_request] (https://deno.land/x/opine@2.0.2/src/router/layer.ts:49:9)
```

**Comment by cmorten** [comment]
@btwiuse and @reggi, apologies for the slow reply on this one!

I upgraded some deps to improve the error logging so could investigate why the `read` module was throwing and in doing so it appears to have fixed the issue! (Something changing between `2.0.2` and `2.3.1` in `opine` it seems).

See https://dash.deno.com/playground/opine-proxy-demo as a working example

Let me know if have more issues, otherwise consider this closed with the release of `3.0.2` 🎉
